server/game/core/gameSteps/AbilityResolver.js

```diff
@@ -75,6 +76,7 @@ class AbilityResolver extends BaseStepWithPipeline {
                 context: this.context
             };
             if (this.context.ability.isCardPlayed()) {
+                resolveTriggersAfter = true;
```

Comment:
As noted in the comment on PlayUnitAction, I don't think this is the spot to control this since whether or not the play card action resolves immediately is contextual (e.g. u-wing play cards are nested, alliance dispatcher is not). If we need to keep it here for now for stuff to work right that's fine, but we should a TODO.

That said, I think from other comments / discussion this might not be needed here.